lib/taskana-core/src/test/java/acceptance/workbasket/query/QueryWorkbasketAccTest.java

```diff
@@ -597,17 +736,63 @@ Stream<DynamicTest> should_SortQueryDesc_When_OrderingByCustomAttribute() {
 
     ThrowingConsumer<WorkbasketCustomField> test =
         customField -> {
-          List<WorkbasketSummary> results =
-              WORKBASKET_SERVICE
-                  .createWorkbasketQuery()
-                  .orderByCustomAttribute(customField, DESCENDING)
-                  .list();
-
-          assertThat(results)
-              .hasSizeGreaterThan(2)
-              .extracting(w -> w.getCustomField(customField))
-              .isSortedAccordingTo(CASE_INSENSITIVE_ORDER.reversed());
+          if (customField != WorkbasketCustomField.CUSTOM_1
+              && customField != WorkbasketCustomField.CUSTOM_2
+              && customField != WorkbasketCustomField.CUSTOM_3
+              && customField != WorkbasketCustomField.CUSTOM_4) {
+
+            Workbasket wb1 = createDefaultWorkbasketWithCustom(customField, "abc123", "NT1234");
+            Workbasket wb2 = createDefaultWorkbasketWithCustom(customField, "def456", "NT2345");
+
+            checkSortCustomDsc(customField);
+
+            WORKBASKET_SERVICE.deleteWorkbasket(wb1.getId());
+            WORKBASKET_SERVICE.deleteWorkbasket(wb2.getId());
+          }
+
+          checkSortCustomDsc(customField);
         };
     return DynamicTest.stream(iterator, c -> "for " + c, test);
   }
+
+  private Workbasket createDefaultWorkbasketWithCustom(
```

Comment:
Can we create Workbaskets with the custom fields 5,6,7,8 in the same way as the Workbaskets with the fields 1,2,3,4? What do you think about it?

Reply:
Where do you use the "createDefaultWorkbasketWithCustom" method?